api/features/import_export/views.py

```diff
@@ -123,3 +129,28 @@ def get_queryset(self) -> QuerySet[FeatureExport]:
         return FeatureExport.objects.filter(environment__in=environment_ids).order_by(
             "-created_at"
         )
+
+
+class FeatureImportViewSet(viewsets.GenericViewSet, mixins.ListModelMixin):
+    serializer_class = FeatureImportSerializer
+    permission_classes = [FeatureImportListPermissions]
+
+    def get_queryset(self) -> QuerySet[FeatureImport]:
+        environment_ids = []
+        user = self.request.user
+
+        for environment in Environment.objects.filter(
+            project_id=self.kwargs["project_pk"],
+        ):
+            if user.is_environment_admin(environment):
+                environment_ids.append(environment.id)
+
+        return FeatureImport.objects.filter(environment__in=environment_ids).order_by(
+            "-created_at"
+        )
+
+
+class FeatureImportRetrieve(viewsets.GenericViewSet, mixins.RetrieveModelMixin):
+    serializer_class = FeatureImportSerializer
+    permission_classes = [FeatureImportRetrievePermissions]
+    queryset = FeatureImport.objects.all()
```

Comment:
Since we're only including one action on each of these viewsets, perhaps we should just use standard `APIView` classes? `ListAPIView` / `RetrieveAPIView` I mean.

Reply:
Thinking about it, I'm not sure why we don't just combine them? I get that in one we have the object to do the permission check, but I don't really see how it's better than just filtering out in the queryset (which would essentially have the same effect as the `get_object_or_404` that I suggested above).

Reply:
From our discussion on slack: Switch to just using the List and used the `ListAPIView` as mentioned.